ApiClient.Tests/Api/EmailApiTest.cs

```diff
@@ -0,0 +1,1346 @@
+        }
+
+        [TestMethod]
+        public void ShouldGetEmailLogsTest()
+        {
+            string expectedFrom = "Jane Doe <jane.doe@somecompany.com";
+            string expectedTo = "john.smith@somedomain.com";
+            string expectedText = "Mail body text";
+            int expectedMessageCount = 1;
+            string expectedMessageId = "MSG-1234";
+            string expectedBulkId = "BULK-1234";
+            DateTimeOffset expectedSentAt = new DateTimeOffset(2021, 9, 2, 9, 57, 56, TimeSpan.FromHours(0));
+            DateTimeOffset expectedDoneAt = new DateTimeOffset(2021, 9, 2, 9, 57, 56, TimeSpan.FromHours(0));
+            string expectedCurrency = "EUR";
+            decimal expectedPricePerMessage = 0.0m;
+            string expectedStatusName = "DELIVERED_TO_HANDSET";
+            int expectedStatusId = 5;
+            string expectedStatusDescription = "Message delivered to handset";
+            int expectedStatusGroupId = 3;
+            string expectedStatusGroupName = "DELIVERED";
+            string expectedChannel = "EMAIL";
+
+            string expectedResponse = $@"
+            {{
+                ""results"": [
+                {{
+                    ""messageId"": ""{expectedMessageId}"",
+                    ""to"": ""{expectedTo}"",
+                    ""from"": ""{expectedFrom}"",
+                    ""text"": ""{expectedText}"",
+                    ""sentAt"": ""{expectedSentAt.ToUniversalTime().ToString(DATE_FORMAT)}"",
+                    ""doneAt"": ""{expectedDoneAt.ToUniversalTime().ToString(DATE_FORMAT)}"",
+                    ""messageCount"": {expectedMessageCount},
+                    ""price"": {{
+                        ""pricePerMessage"": {expectedPricePerMessage.ToString("N", CultureInfo.InvariantCulture)},
+                        ""currency"": ""{expectedCurrency}""
+                    }},
+                    ""status"": {{
+                        ""groupId"": {expectedStatusGroupId},
+                        ""groupName"": ""{expectedStatusGroupName}"",
+                        ""id"": {expectedStatusId},
+                        ""name"": ""{expectedStatusName}"",
+                        ""description"": ""{expectedStatusDescription}""
+                    }},
+                    ""channel"": ""{expectedChannel}"",
+                    ""bulkId"": ""{expectedBulkId}""
+                }}
+                ]
+            }}";
+
+            SetUpGetRequest(EMAIL_REPORTS_ENDPOINT, expectedResponse, 200);
+
+            var sendEmailApi = new EmailApi(configuration);
+
+            EmailLogsResponse response = sendEmailApi.GetEmailLogs(
+                bulkId: expectedBulkId,
+                messageId: expectedMessageId,
+                limit: 2
+            );
+
+            Assert.AreEqual(response.Results.Count, 1);
+
+            Assert.AreEqual(response.Results[0].From, expectedFrom);
```

Comment:
Is it possible to serialize gathered response and compare it with expected response as two JSONs?